core/src/main/groovy/noe/tomcat/configure/TomcatConfigurator.groovy

```diff
@@ -39,6 +40,11 @@ class TomcatConfigurator {
 
   // - Methods ----------------------------------------------------------------------------------------
 
+  TomcatConfigurator keepDefaultAddressAndPort(boolean flag) {
```

Comment:
I would rather see `set` keyword there instead of "overloading" variable.

Reply:
me too, nice catch - updated